Client/Frontend/Autofill/CreditCard/CreditCardSettingsViews+Model/CreditCardTableViewController.swift

```diff
@@ -0,0 +1,194 @@
+
+        viewModel.didUpdadteCreditCards = { [weak self] in
+            self?.reloadData()
+        }
+
+        NSLayoutConstraint.activate([
+            toggleSwitchContainer.topAnchor.constraint(equalTo: view.topAnchor),
+            toggleSwitchContainer.heightAnchor.constraint(equalToConstant: 40),
+            toggleSwitchContainer.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            toggleSwitchContainer.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+
+            toggleSwitch.centerYAnchor.constraint(
+                equalTo: toggleSwitchContainer.centerYAnchor),
+            toggleSwitch.trailingAnchor.constraint(equalTo: toggleSwitchContainer.trailingAnchor, constant: -16),
+
+            toggleSwitchLabel.centerYAnchor.constraint(
+                equalTo: toggleSwitchContainer.centerYAnchor),
+            toggleSwitchLabel.leadingAnchor.constraint(equalTo: toggleSwitchContainer.leadingAnchor, constant: 16),
+            toggleSwitchLabel.heightAnchor.constraint(equalToConstant: 18),
+
+            toggleSwitchContainerLine.heightAnchor.constraint(equalToConstant: 0.7),
+            toggleSwitchContainerLine.leadingAnchor.constraint(
+                equalTo: toggleSwitchContainer.leadingAnchor, constant: 10),
+            toggleSwitchContainerLine.trailingAnchor.constraint(
+                equalTo: toggleSwitchContainer.trailingAnchor, constant: 10),
+            toggleSwitchContainerLine.bottomAnchor.constraint(
+                equalTo: toggleSwitchContainer.bottomAnchor),
+
+            savedCardsTitleLabel.topAnchor.constraint(equalTo: toggleSwitchContainer.bottomAnchor, constant: 25),
+            savedCardsTitleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            savedCardsTitleLabel.heightAnchor.constraint(equalToConstant: 13),
+            savedCardsTitleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+
+            tableView.topAnchor.constraint(equalTo: savedCardsTitleLabel.bottomAnchor, constant: 8),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        ])
+
+        view.bringSubviewToFront(tableView)
+    }
+
+    func applyTheme(theme: Theme) {
+        toggleSwitchContainerLine.backgroundColor = theme.colors.borderPrimary
+        toggleSwitchContainer.backgroundColor = theme.colors.layer2
+        tableView.backgroundColor = .clear// theme.colors.layer2
+        toggleSwitch.onTintColor = theme.colors.actionPrimary
+        view.backgroundColor = theme.colors.layer1
+    }
+
+    @objc private func autofillToggleTapped() {
+        viewModel.updateToggle()
+        updateToggleValue(value: viewModel.autofillCreditCardStatus)
+    }
+
+    func reloadData() {
+        tableView.reloadData()
+    }
+
+    func updateToggleValue(value: Bool) {
```

Comment:
This method is only called once. I think we can add the one line of it to where it's called and remove it.